src/topp/IDFileConverter.cpp

```diff
@@ -170,6 +176,9 @@ class TOPPIDFileConverter :
     registerFlag_("ignore_proteins_per_peptide", "[Sequest only] Workaround to deal with .out files that contain e.g. \"+1\" in references column,\n"
                                                  "but do not list extra references in subsequent lines (try -debug 3 or 4)", true);
     registerStringOption_("scan_regex", "<expression>", "", "[Mascot, pepXML, Percolator only] Regular expression used to extract the scan number or retention time. See documentation for details.", false, true);
+    registerFlag_("add_spectrum_references", "Adds the spectrum files spectrum ID reference to each PeptideIdentification. Warning: works only with mz_file set", true);
+    registerDoubleOption_("add_ionmatch_annotation", "<tolerance value in Da>", 0, "Adds IonMatch statistics to each PeptideHit if tolerance value > 0 e.g. 0.5. Warning: works only with mz_file set, might be time and memory laborious!", false, true);
```

Comment:
Better: "Tolerance in Da for ion match annotation. If set to a positive value, ion match statistics are added to each peptide hit. This requires 'mz_file' to be set, and might be time/memory intensive."
Again: "PeptideHit" -> OpenMS class; "peptide hit" -> actual entity